src/formatDate.js

```diff
@@ -8,7 +8,42 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separatorFrom = fromFormat.pop();
```

Comment:
Using `pop` mutates the original `fromFormat` array, which is not a good practice as it may lead to unexpected behavior if the array is used elsewhere after calling this function. Instead, you could access the last element directly without mutating the array.